.github/workflows/package.yml

```diff
@@ -58,20 +62,35 @@ jobs:
           git submodule foreach --recursive git clean -ffdx
           git submodule foreach --recursive git reset --hard
 
-      - name: check bash
-        run: yarn run check:bash:ci
-
       - uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node }}
-          architecture: ${{ matrix.host }}
 
-      - name: Add yarn
-        if: matrix.os == 'macos-13' || matrix.os == 'macos-14'
-        run: npm install -g yarn
+      - uses: actions/cache@v4
+        id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
```

Comment:
Is the comment needed? You have the conditional below?

Reply:
Removed.